flows/atd-executive-dashboard/row_active_permits.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+"""
+Name: ATD Performance Dashboard: ROW Active Permits Logging
+Description: Repo: https://github.com/cityofaustin/atd-executive-dashboard Stores the current number of active ROW permits
+    in a Socrata dataset.
+
+Create Deployment:
+$ prefect deployment build flows/atd-executive-dashboard/row_active_permits.py:main \
+    --name "ATD Performance Dashboard: ROW Active Permits Logging" \
+    --pool atd-data-03 \
+    --cron "0 8 * * 1" \
+    -q default \
+    -sb github/ch-amanda-queries \
+    -o "deployments/row_active_permits.yaml"
+    --description Description: Repo: https://github.com/cityofaustin/atd-executive-dashboard Stores the current number of active ROW permits in a Socrata dataset.
+ 
+$ prefect deployment apply deployments/row_active_permits.yaml
+"""
+
+import os
+
+import docker
+import prefect
+from datetime import datetime, timedelta
+
+# Prefect
+from prefect import flow, task, get_run_logger
+from prefect.blocks.system import JSON
+
+
+# Docker settings
+docker_env = "latest"
+docker_image = f"atddocker/atd-executive-dashboard"
+
+
+@task(
+    name="get_env_vars",
+    retries=10,
+    retry_delay_seconds=timedelta(seconds=15).seconds,
```

Comment:
This is fine - but it got me thinking. I wrote this [stilly little replit](https://replit.com/@frankhereford1/AvariciousOilyFlashmemory#main.py) to see what that `timedelta().seconds` resolves to, and it's an `int`. Do you think just `retry_delay_seconds=15,` might be a more clear way to state this?